app/src/components/InputElement/InputElement.styles.js

```diff
@@ -66,3 +66,16 @@ export const StyledInputField = styled.input`
     margin-right: 1rem;
   }
 `;
+
+export const StyledLinksList = styled.div`
+  @media (min-width: 350px) {
```

Comment:
i thought this breakpoint will be removed ;)

Reply:
fixed